lib/request.js

```diff
@@ -92,9 +92,14 @@ function Request (options) {
 
   const parsedURL = parseURL(options.url || options.path, options.query)
 
+  /**
+   * https://github.com/nodejs/node/blob/75b68c6e4db515f76df73af476eccf382bbcb00a/lib/_http_common.js
```

Comment:
The link is not explaining anything. 

You mean this line?
https://github.com/nodejs/node/blob/75b68c6e4db515f76df73af476eccf382bbcb00a/lib/_http_common.js#L94

Reply:
Maybe just remove the comment :/

Reply:
ok